src/wp-includes/html-api/class-wp-html-open-elements.php

```diff
@@ -101,6 +101,51 @@ public function set_push_handler( Closure $handler ) {
 		$this->push_handler = $handler;
 	}
 
+	/**
+	 * Returns the name of the node at the nth position on the stack
+	 * of open elements, or `null` if no such position exists.
+	 *
+	 * Note that this uses a 1-based index, which represents the
+	 * "nth item" on the stack, counting from the top, where the
+	 * top-most element is the 1st, the second is the 2nd, etc...
+	 *
+	 * @todo Skip over "marker" entries on the stack, if appropriate.
+	 *
+	 * @since 6.7.0
+	 *
+	 * @param int $nth Retrieve the nth item on the stack, with 1 being
+	 *                 the top element, 2 being the second, etc...
+	 * @return string|null Name of the node on the stack at the given location,
+	 *                     or `null` if the location isn't on the stack.
+	 */
+	public function at( int $nth ) {
```

Comment:
One of the methods has a PHP return type hint, so seems like this should be added to the others as well. For example:

```suggestion
	public function at( int $nth ): ?string {
```